larky/src/main/resources/vendor/vgs/nts_helpers.star

```diff
@@ -0,0 +1,25 @@
+load("@stdlib//larky", larky="larky")
+load("@vendor//jsonpath_ng", jsonpath_ng="jsonpath_ng")
+load("@vgs//nts", "nts")
+
+
+def render(input, pan, exp_month=None, exp_year=None, cryptogram_value=None, cryptogram_eci=None):

```

Comment:
I like this a lot. This way, in more complex (non-JSON) integrations, we can just call "nts.get_netwok_token(pan)" to get the exact data and fill in the fields as needed.